next.config.js

```diff
@@ -1,7 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: false,
-  swcMinify: false,
```

Comment:
What is this needed for? This will be set to true with no way to change it in the next next.js release.

Reply:
I think it was in order to keep source map support working correctly. Maybe they fixed that?

Reply:
Maybe :D